src/main/java/org/stellar/anchor/paymentservice/circle/model/CirclePaymentStatus.java

```diff
@@ -0,0 +1,41 @@
+package org.stellar.anchor.paymentservice.circle.model;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Getter;
+import org.stellar.anchor.paymentservice.Payment;
+
+@Getter
+public enum CirclePaymentStatus {
```

Comment:
Do we want to store any kind of error message or result code? When I use Circle's APIs and get failed transactions, there is usually an error code associated with it.

Reply:
The error message is being parsed from the transfer response and included in the Payment object:
https://github.com/stellar/java-stellar-anchor-sdk/blob/8f5f0bc22a75745b485c5032d32c5f8b8599cc46/src/main/java/org/stellar/anchor/paymentservice/circle/model/CircleTransfer.java#L20-L32

Whoever has access to the Payment object (another application or an API layer) will have access to this information.